server/tests/test_init_session.rs

```diff
@@ -2,15 +2,15 @@ use std::convert::TryFrom;
 
 use std::ops::DerefMut;
 use ya_client_model::NodeId;
+use ya_relay_client::ClientBuilder;
+use ya_relay_core::session::SessionId;
 use ya_relay_proto::proto;
 use ya_relay_server::testing::server::init_test_server;
-use ya_relay_server::testing::ClientBuilder;
-use ya_relay_server::SessionId;
 
 #[serial_test::serial]
 async fn test_query_self_node_info() -> anyhow::Result<()> {
     let wrapper = init_test_server().await.unwrap();
-    let client = ClientBuilder::from_server(&wrapper.server)
+    let client = ClientBuilder::from_url(wrapper.server.inner.url.clone())
```

Comment:
It's not so elegant

Reply:
its more elegant then a client that only knows about the server object to read a single property :)